ansible_collections/arista/avd/roles/eos_designs/docs/L3 Edge.md

```diff
@@ -136,6 +136,7 @@ Make sure to configure the variables in a group_vars file covering all devices m
           mtu: <int>
           bfd: <bool>
           ptp:
+            enable: <bool>
```

Comment:
@nielsjlarsen - running pre-commit locally will remove this line and the next in the doc. is it expected?

Reply:
```
--- a/ansible_collections/arista/avd/roles/eos_designs/docs/L3 Edge.md
+++ b/ansible_collections/arista/avd/roles/eos_designs/docs/L3 Edge.md
@@ -136,7 +136,6 @@ Make sure to configure the variables in a group_vars file covering all devices m
           mtu: <int>
           bfd: <bool>
           ptp:
-            enable: <bool>
           qos_profile: <str>
           macsec_profile: <str>
           port_channel:
@@ -174,7 +173,6 @@ Make sure to configure the variables in a group_vars file covering all devices m
           mtu: <int>
           bfd: <bool>
           ptp:
-            enable: <bool>
           qos_profile: <str>
           macsec_profile: <str>
           port_channel:
```